fcrepo4/fcrepo4
# update version of ocfl-java

**JIRA Ticket**: https://jira.lyrasis.org/browse/FCREPO-3371

# What does this Pull Request do?

1. Upgrades to the latest version of ocfl-java.
2. Changes the ocfl storage layout
3. Stopped encoding logical paths on windows -- ocfl-java is configured to produce safe mappings internally

# How should this be tested?

1. Create a resource
2. Examine the ocfl repository and see that the objects are now store in a hashed truncated n-tuple layout, based on the sha256sum of the ocfl object id

# Interested parties
@fcrepo4/committers
